lnwallet/btcwallet/btcwallet.go

```diff
@@ -710,6 +712,60 @@ func (b *BtcWallet) ListAccounts(name string,
 	return res, nil
 }
 
+// CurrentNumAnchorChans returns the current number of non-private anchor
+// channels the wallet should be ready to fee bump if needed.
+func (b *BtcWallet) CurrentNumAnchorChans() (int, error) {
```

Comment:
Hmm, I don't really like this code move since it requires the btc wallet to know about the channel DB, which is a clear violation of the responsibility boundaries...
I think the better alternative is probably to add the `CurrentNumAnchorChans()` as a function argument to the `lnrpc/walletrpc/config_active.go` file. See the example in `lnrpc/peersrpc/config_active.go`. That probably results in the smallest diff.